src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -533,6 +556,41 @@ const getOpenscap = (state: RootState): OpenScap | undefined => {
   return undefined;
 };
 
+const mapUserInfoToUser = (userInfo: {
+  userName: string;
+  password: string;
+  sshKey: string;
+}): User => {
+  return {
+    name: userInfo.userName,
+    password: userInfo.password || '',
+    ssh_key: userInfo.sshKey || '',
+  };
+};
+
+const getUserInfo = (state: RootState): User | undefined => {
+  const userInfo = state.wizard.userInfo;
+
+  if (
+    (userInfo.userName && userInfo.password) ||
+    (userInfo.userName && userInfo.sshKey)
+  ) {
+    return mapUserInfoToUser(userInfo);
+  }
+
+  return undefined;
+};
+
+const getUsers = (state: RootState): User[] => {
+  const currentUser = getUserInfo(state);
+
+  if (currentUser) {
+    return [...state.wizard.users, currentUser];
+  }
+
+  return state.wizard.users;
+};
```

Comment:
It's a bit hard for me to get oriented with the `users` and `userInfo`, but what we want to do here is take the values from the store and shape them to fit the request.

So we should probably check if there are any users store in the `users` array in the store and if there are any items, we will want to iterate through them and for each push smth like this to the `users` array of the request:
```
{
  name: currentUser.name,
  password: currentUser.password || '',
  ssh_key: currentUser.ssh_key || '',
}
```